src/scale.c

```diff
@@ -38,6 +23,7 @@ avifBool avifImageScale(avifImage * image,
 #if defined(__clang__)
 #pragma clang diagnostic pop
 #endif
+#endif
```

Comment:
This was not the case at head but please comment the name of the conditional define:
```c
#endif
#endif // AVIF_LIBYUV_ENABLED
```